config/clusters/cloudbank/csum.values.yaml

```diff
@@ -39,13 +39,10 @@ jupyterhub:
           https://cma-shibboleth.csum.edu/idp/shibboleth:
             username_derivation:
               username_claim: "email"
-            # allow_all is a partial authorization, username_pattern is enforced also
             allow_all: true
           http://google.com/accounts/o8/id:
             username_derivation:
               username_claim: "email"
-            # allow_all is a partial authorization, username_pattern is enforced also
-            allow_all: true
```

Comment:
This hub didn't have username_pattern configured, so declaring `allow_all` led to authorizing all google users. Without having `allow_all`, only google users with certain usernames listed in `admin_users` or similar would be allowed.